okhttp/src/jvmTest/java/okhttp3/CipherSuiteTest.java

```diff
@@ -111,7 +111,7 @@ public class CipherSuiteTest {
         forJavaName("TLS_FAKE_NEW_CIPHER"));
   }
 
-  @Test public void applyIntersectionRetainsSslPrefixes() throws Exception {
+  @Test public void applyIntersectionRetainsTlsPrefixes() throws Exception {
```

Comment:
What is happening here?

Reply:
Yep, it needs to switch because it follows a different set.